### DISCUSSION TITLE
Profile picture/logo receives 403

### DISCUSSION THREAD

**Discussion by mmnaseri** [outline/outline#2765] [Self Hosting]
**To Reproduce**
1. Go to settings.
2. Add a profile picture, or a logo for the app.
3. Refresh.
4. See that the pictures don't load.
5. Open the image in a new tab.
6. See the 403.

**Expected behavior**
Expected the pictures to show up.

**Outline (please complete the following information):**
- Install: self hosted
- Version: 28ab340 (v0.60.1)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox
 - Version: 94

The details of my setup:
- Configured with MinIO instance.
- Log-in via Keycloak.

No other image upload exhibits this behavior and everything works just fine. It is just profile pictures and the overall logo that cause problem. `docker service logs outline` doesn't produce any output for this image URL. The image URL has the following format:

https://my-domain-address/outline-data/public/422e2212-eff4-4f53-965e-674a5234570c/c94ed799-de07-4975-b91f-41ccee691ecc/image-name.png

**Comment by v-marinkov** [comment]
I just experienced the same issue, just a 400 instead of 403. Is there a solution for it?

**Reply by michaelhofer** [reply]
We have the same issue, but it only happens when the S3 bucket (minio in our case) is NOT set to public.

**Reply by philipgierszal** [reply]
I am having the same problem, where outline uses the path https://minio.MYDOMAIN.com/BUCKETNAME/public/RESOURCE-ID
so outline forces minio to be public, although AWS_S3_ACL is set to private. Any ideas how to change that?

**Reply by Limezy** [reply]
Now I finally understand, thanks for the explanation !

**Comment by vincentbaeten** [comment]
Goto buckets -> manage -> "Access rules". Then add an access rule for both the public and avatars folder. This way the rest of the bucket can remain private.

<img width="958" alt="Screen Shot 2022-02-19 at 4 48 56 PM" src="https://user-images.githubusercontent.com/37792455/154808127-59749123-27fa-4f20-a9ac-6f8fc211afef.png">

**Reply by v-marinkov** [reply]
Do you know what the equivalent action for AWS S3 would be?
